packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -2039,6 +2046,854 @@ contract OPContractsManagerInteropMigrator is OPContractsManagerBase {
     }
 }
 
+/// @title OPContractsManagerV2
+/// @notice OPContractsManagerV2 is an enhanced version of OPContractsManager that unifies all
+///         existing OPContractsManager functionality into a single contract and codepath.
+contract OPContractsManagerV2 is OPContractsManagerBase {
+    /// @notice Configuration for the FaultDisputeGame.
+    struct FaultDisputeGameConfig {
+        Claim absolutePrestate;
+    }
+
+    /// @notice Configuration for the PermissionedDisputeGame.
+    struct PermissionedDisputeGameConfig {
+        Claim absolutePrestate;
```

Comment:
Do we really need to set a prestate for the permissioned game? We set it to something like `hex"dead"` when we deploy new chains, can we simplify things to just drop this param and always hardcode it as `dead` for all permissioned games?